frontend/src/api/appwriteApi.js

```diff
@@ -201,6 +195,45 @@ let api = {
 		});
 	},
 
+	/**
+	 * Storage API
+	 **/
+	imageToFileID(imageID) {
+		return imageID && `img${imageID}`;
+	},
+
+	async loadImageFromDatabase(imageID) {
+		const fileID = this.imageToFileID(imageID);
```

Comment:
For consistency, we should use the same way of declaring functions throughout the api file imo.

Reply:
Do you mean using `this` and not using the lambda expression syntax?